kafka-ui-e2e-checks/src/main/resources/META-INF/services/org.junit.platform.launcher.TestExecutionListener

```diff
@@ -0,0 +1 @@
+extensions.QaseExtension
```

Comment:
I don't feel like we need to commit any META-INF contents

Reply:
it's listener of Junit5